src/registrar/models/very_important_person.py

```diff
@@ -0,0 +1,36 @@
+from django.db import models
+
+from .utility.time_stamped_model import TimeStampedModel
+
+
+class VeryImportantPerson(TimeStampedModel):
+
+    """"""
+
+    email = models.EmailField(
+        null=True,
+        blank=True,
+        help_text="Email",
+        db_index=True,
+    )
+
+    user = models.ForeignKey(
+        "registrar.User",
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="verifiedby_user",
+    )
+
+    notes = models.TextField(
+        null=True,
+        blank=True,
+        help_text="Notes",
+    )
+
+    def __str__(self):
+        try:
+            if self.email:
+                return self.email
+        except Exception:
+            return ""
```

Comment:
```suggestion
    def __str__(self):
        return self.email # Don't commit this directly suggestion, spacing may be off
```
This should just return the email, but do some sort of error handling step to catch edge cases -- such as like if not isinstance(self.email, str) -- if that is what you are encountering. I don't think you would need that though, since models.emailfield has built in validation I think